service/qos/standard_service_level_distributed_data_accessor.hh

```diff
@@ -22,6 +22,9 @@ private:
     db::system_distributed_keyspace& _sys_dist_ks;
 public:
     standard_service_level_distributed_data_accessor(db::system_distributed_keyspace &sys_dist_ks);
+    virtual future<> start_service_level_operation() override;
+    virtual void abort_service_level_operation() override;
```

Comment:
A nicer design is to return a transaction object from start(). The guard object then has commit() and abort()  methods (and its destructor can abort too). This prevents an exception from having a start() that is not paired with a commit or abort.

Reply:
Redesigned, so now there is a service_level_transaction object

Reply:
Changed to use `group0_guard` from cql-statement